common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -558,6 +600,8 @@ void remove() {
             HashedWheelBucket bucket = this.bucket;
             if (bucket != null) {
                 bucket.remove(this);
+            } else {
```

Comment:
I would prefer to remove it as I think this is an "implementation" detail and also there is no need to introduce overhead for users that not want any bound